Rename RitAssist component to FleetGO

wimhaanstra opened the pull request:
The company has been renamed from RitAssist to FleetGO, so this component needed to get renamed.

## Breaking Change:

The integration has been renamed from `ritassist` to `fleetgo`. Users of this integration will need to update their `device_tracker` config section to use the new name.

## Description:
The component renamed itself from Rit Assist to FleetGO this year and the component was updated to reflect the new name

**Pull request with documentation for [home-assistant.io](https://github.com/home-assistant/home-assistant.io) (if applicable):** home-assistant/home-assistant.io#9591

## Example entry for `configuration.yaml` (if applicable):
```yaml

```

## Checklist:
  - [x] The code change is tested and works locally.
  - [x] Local tests pass with `tox`. **Your PR cannot be merged unless tests pass**
  - [x] There is no commented out code in this PR.
  - [x] I have followed the [development checklist][dev-checklist]

If user exposed functionality or configuration variables are added/changed:
  - [x] Documentation added/updated in [home-assistant.io](https://github.com/home-assistant/home-assistant.io)

If the code communicates with devices, web services, or third-party tools:
  - [x] [_The manifest file_][manifest-docs] has all fields filled out correctly. Update and include derived files by running `python3 -m script.hassfest`.
  - [x] New or updated dependencies have been added to `requirements_all.txt` by running `python3 -m script.gen_requirements_all`.
  - [x] Untested files have been added to `.coveragerc`.

If the code does not interact with devices:
  - [x] Tests have been added to verify that the new code works.

[dev-checklist]: https://developers.home-assistant.io/docs/en/development_checklist.html
[manifest-docs]: https://developers.home-assistant.io/docs/en/creating_integration_manifest.html

Adminiuga committed:
Bump ZHA dependencies. (#24637)

balloob committed:
Warn when user tries run custom config flow (#24657)

fronzbot committed:
Upgrade blinkpy==0.14.1 for startup bugfix (#24656)

amelchio committed:
Update LIFX brightness during long transitions (#24653)

andrewsayre committed:
Bump pysmartthings (#24659)

foreign-sub committed:
Fix AttributeError: 'NoneType' object has no attribute 'group' with sytadin component (#24652)

* Fix AttributeError: 'NoneType' object has no attribute 'group'

* Update sensor.py

scop committed:
braviatv, nmap_tracker: use getmac for getting MAC addresses (#24628)

* braviatv, nmap_tracker: use getmac for getting MAC addresses

Refs https://github.com/home-assistant/home-assistant/pull/24601

* Move getmac imports to top level

sfjes committed:
Fix downloader_download_failed event not firing for HTTP response errors (#24640)

balloob committed:
Updated frontend to 20190620.0

pvizeli committed:
Update azure-pipelines-wheels.yml for Azure Pipelines

mvn23 committed:
Multiple devices support for opentherm_gw (#22932)

* Breaking change: Rewrite opentherm_gw to add support for more than one OpenTherm Gateway.
Breaks config layout and child entity ids and adds a required parameter to all service calls (gateway_id).

* Add schema and parameter description for service opentherm_gw.reset_gateway.

* Add optional name attribute in config to be used for friendly names.
Fix bugs in binary_sensor and climate platforms.

* pylint fixes

* Remove unused variables.

* Update manifest.json, remove REQUIREMENTS from .py file

* Update CODEOWNERS

* Address issues that were brought up (requested changes):
- Move imports to module level
- Change certain functions from async to sync
- Move constants to const.py (new file)
- Call gateway setup from outside of __init__()
- Move validation of monitored_variables to config schema

* Address requested changes:
- Make module imports relative
- Move more functions from async to sync, decorate with @callback where necessary
- Remove monitored_variables option, add all sensors by default

pvizeli committed:
Update azure-pipelines-wheels.yml for Azure Pipelines

rodripf committed:
Vlc telnet (#24290)

* Vlc telnet first commit

First functional version, remains to add more functionality.

* New functions added and bugfixes

* Compliance with dev checklist

* Compliance with dev checklist

* Compliance with pydocstyle

* Removed unused import

* Fixed wrong reference for exception

* Module renamed

* Fixed module rename in other

* Fixed wrong reference for exception


Module renamed


Fixed module rename in other

* Update homeassistant/components/vlc_telnet/media_player.py

Accepted suggestion by @OttoWinter

Co-Authored-By: Otto Winter <otto@otto-winter.com>

* Update homeassistant/components/vlc_telnet/media_player.py

Accepted suggestion by @OttoWinter

Co-Authored-By: Otto Winter <otto@otto-winter.com>

* Update homeassistant/components/vlc_telnet/media_player.py

Accepted suggestion by @OttoWinter

Co-Authored-By: Otto Winter <otto@otto-winter.com>

* Update homeassistant/components/vlc_telnet/media_player.py

Accepted suggestion by @OttoWinter

Co-Authored-By: Otto Winter <otto@otto-winter.com>

* Suggestions by @OttoWinter

+Manage error when the VLC dissapears to show status unavailable.

* Removed error log, instead set unavailable state

* Changes suggested by @pvizeli

-Import location
-Use of constants

* Implemented available method

* Improved available method

balloob committed:
Clean up Google Config (#24663)

* Clean up Google Config

* Lint

* pylint

* pylint2

balloob committed:
Clean up Google Config (#24663)

* Clean up Google Config

* Lint

* pylint

* pylint2

pvizeli committed:
Prefere binary with wheels (#24669)

bachya committed:
Add device class support for Ambient PWS sensors (#24677)

balloob committed:
Bumped version to 0.95.0b1

pvizeli committed:
Update azure-pipelines-release.yml for Azure Pipelines

zewelor committed:
Improve autodiscovered yeelights model detection (#24671)

* Improve autodiscovered yeelights model detection

* Lint fixes

* Logger warn fix

thomasloven committed:
Allow extra js modules to be included in frontend (#24675)

* Add extra_module_url and extra_module_url_es5 to frontend options

* Address review comments

bachya committed:
Add RainMachine device classes where appropriate (#24682)

bachya committed:
Change Ambient solar radiation units to lx (#24690)

StevenLooman committed:
Upgrade to async_upnp_client==0.14.10 and increase search timeout (#24685)

scop committed:
Upgrade mypy to 0.710 (#24666)

* Upgrade mypy to 0.710

* Address mypy 0.710 errors

Swamp-Ig committed:
Template: Expand method to expand groups, and closest as filter (#23691)

* Implement expand method

* Allow expand and closest to be used as filters

* Correct patch

* Addresses review comments

jmw6773 committed:
Add aml_thermal label (#24665)

Added label for the CPU Temperature for AmLogic ARM chips.

alengwenus committed:
Corrected number of default LCN segment coupler scan tryouts (#24678)

* Bump to pypck==0.6.2

* Set default segment coupler scan tryouts to 0

amelchio committed:
Fix time expression parsing (#24696)

fabaff committed:
Upgrade discord.py to 1.2.2 (#24695)

fabaff committed:
Upgrade youtube_dl to 2019.06.08 (#24692)

dmulcahey committed:
ZHA fix device type mappings (#24699)

fabaff committed:
Upgrade Sphinx to 2.1.2 (#24693)

cgtobi committed:
Bump version pyatmo to 2.0.1 (#24703)

cgtobi committed:
Bump version pyatmo to 2.0.1 (#24703)

squishykid committed:
Solax update 0.1.0 (#24708)

* Update to solax 0.0.6

* Library version 0.1.0

ktnrg45 committed:
Fix PS4 entities with shared host not updating and latency with multiple connections (#24642)

* correct assume info call

* 0.8.4

* 0.8.4

* 0.8.4

* 0.8.5

* 0.8.5

* 0.8.5

* revert condition

dmulcahey committed:
Update ZHA dependencies (#24718)

* update deps and remove legacy constants bridge

* run deps script and fix test import

dmulcahey committed:
Update ZHA dependencies (#24718)

* update deps and remove legacy constants bridge

* run deps script and fix test import

balloob committed:
Update owner stream integration

2sheds committed:
Extend websocket method usage to port 8002 in Samsung TV media player (#24716)

pvizeli committed:
Add initial support for remote dev container (#24681)

* Add initial support for remote container

* Use constrain

Commit:
Expose ports 8123, 8300 and 51827 in Dockerfile (#24389)

cgtobi committed:
Bump pyatmo to v2.1.0 (#24724)

cgtobi committed:
Bump pyatmo to v2.1.0 (#24724)

endor-force committed:
Add Trafikverket train component (#23470)

* Added Trafikverket train component

* Updated manifest with proper name and codeowner

* Updated requirements and manifest

* Updated CODEOWNERS

* Corrected requirements

* Added trafikverket_train/sensor.py to .coveragerc

* Added error handling and log if API call fails

* Corrected styles, removed dev log, improved validation

* Method calls to async_update(), improved error handling

* Minor cleanup/reorg for effeciency

* Added station cache and corrected to fit standards

* Simplified trainstop id  and cleaned up dict.get

* Corrected mistake after change from dict to array

* Change device class to timestamp

pnbruckner committed:
Add show_as_state options to Life360 (#24725)

balloob committed:
Updated frontend to 20190624.0

balloob committed:
Update translations

balloob committed:
Bumped version to 0.95.0b2

squishykid committed:
Add 'unique_id' Property to Inverter Sensors (#24707)

* Option to change sensor names

* Python 3.5 compatibility

* Oops

* Get serial number at start

* Remove config opportunity

* Oops comma

* Changes from review

* Check yourself before you commit.

evanjd committed:
Save cached logi_circle tokens in config folder (#24726)

Instead of the working directory, which it's doing currently. Matches pattern observed on Abode, Ring, Skybell integrations.

dmulcahey committed:
Remove device and entity registry entries when removing a ZHA device (#24369)

* cleanup when device is removed

fixes

* cleanup

cnrd committed:
Add support for sensor state STATE_UNAVAILABLE (#24641)

* Fixed integration with ESPhome, which caused an error if ESPhome did not update fast enough on startup

* Set state to problem if sensor is unavailable

* Fix line length.

Adminiuga committed:
Update ZHA dependencies. (#24736)

balloob committed:
AdGuard to update entry (#24737)

amelchio committed:
Update pysonos to 0.0.17 (#24740)

balloob committed:
Bumped version to 0.95.0b3

MartinHjelmare committed:
Fix locative device update (#24744)

* Add a test for two devices

* Fix locative updating all devices

* Add a guard clause that checks if correct device is passed.

balloob committed:
Improve Alexa error handling (#24745)

balloob committed:
Improve Alexa error handling (#24745)

balloob committed:
Updated frontend to 20190624.1

balloob committed:
Bumped version to 0.95.0b4

andre-richter committed:
Add integration for Vallox Ventilation Units (#24660)

* Add integration for Vallox Ventilation Units.

* Address review comments #1

* Address review comments #2

* Replace IOError with OSError.

* Bump to fixed version of vallox_websocket_api.

cgtobi committed:
Handle timeouts gracefully (#24752)

balloob committed:
Ignore duplicate tradfri discovery (#24759)

* Ignore duplicate tradfri discovery

* Update name

andre-richter committed:
Vallox: Fix missing hass member (#24753)

Emilv2 committed:
Remove obsolete comments in Dockerfile (#24748)

relevant lines were removed in e49b970665a8d798deab86cc7c836830e9984113

alain57 committed:
fix an error in the description (#24735)

Danielhiversen committed:
Return correct name for met.no (#24763)

johntdyer committed:
Update Waze route dependency to 0.10  (#24754)

* Update manifest.json

Update waze calculator to 0.10, this was supposed to have been done in #22428 but was missed. See discussion [here](https://community.home-assistant.io/t/waze-travel-time-update/50955/201)

* Update requirements_all.txt

Matte23 committed:
Add support for IPP Printers to the CUPS integration (#24756)

* Add support for IPP Printers to the CUPS integration

* Fixed lint error

* Addressed comments, removed redundant check

* Simplified check, improved code readability

Adminiuga committed:
Enhancement/zha model manuf (#24771)

* Cleanup ZHA entities model and manufacturer usage.
Zigpy includes manufacturer and model as attributes of a zigpy
Device class, which simplifies handling of manufacturer and/or model
derived properties for the ZHA platform.

* Sort ZHA imports.
* Lint.

balloob committed:
Updated frontend to 20190626.0

balloob committed:
Bumped version to 0.95.0

balloob committed:
Entity to handle updates via events (#24733)

* Entity to handle updates via events

* Fix a bug

* Update entity.py

andre-richter committed:
Upgrade vallox to async client API (#24774)

pnbruckner committed:
Fix life360 exception when no location provided (#24777)

Commit:
Pubnub to 1.0.8 (#24781)

scop committed:
Upgrade pytest to 4.6.3 (#24782)

* Upgrade pytest to 4.6.3

https://docs.pytest.org/en/latest/changelog.html#pytest-4-6-2-2019-06-03
https://docs.pytest.org/en/latest/changelog.html#pytest-4-6-3-2019-06-11

* Make litejet switch test work with pytest 4.6.3

Essentially reverts the corresponing change that was made for pytest
4.2 compatibility.

balloob committed:
Catch uncaught Alexa error (#24785)

balloob committed:
Bumped version to 0.95.1

h3ndrik committed:
Add time delta option when searching for deutsche_bahn connections (#24600)

* Add time delta option when searching for connections

Add another option 'in' to search for upcoming connections in the future.
Handy if you need a few minutes to get to the train station and need to add that to the queried departure time.

* correct style errors

* rename new option

* rename new option (2/2)

* add offset correctly

pvizeli committed:
Update azure-pipelines-release.yml for Azure Pipelines (#24800)

balloob committed:
Make sure entity config is never none (#24801)

dmulcahey committed:
Bump ZHA quirks module (#24802)

* bump quirks version

* bump version - mija magnet

dreed47 committed:
Zestimate fix for issue #23837 (#23838)

* Zestimate fix for issue #23837

removed references to MIN_TIME_BETWEEN_UPDATES
and replaced with SCAN_INTERVAL

* Zestimate fix for issue #23837

removed references to MIN_TIME_BETWEEN_UPDATES
and replaced with SCAN_INTERVAL

pnbruckner committed:
Fix another Life360 bug (#24805)

andersonshatch committed:
Use climate device's target temp step value (#24804)

andersonshatch committed:
Use step from tado rather than assuming 0.1 (#24807)

balloob committed:
Updated frontend to 20190627.0

GrandNewbien committed:
Changed default encoding to mp3 (#24808)

cgtobi committed:
Fix netatmo weatherstation setup error  (#24788)

* Check if station data exists and reduce calls

* Fix module names list

* Add warning

* Remove dead code

balloob committed:
Add developer tools panel (#24812)

balloob committed:
Fix calling empty script turn off (#24827)

balloob committed:
Bumped version to 0.95.2

luukd committed:
Add template vacuum support (#22904)

* Add template vacuum component

* Fix linting issues

* Make vacuum state optional

* Fix pylint issues

* Add context to template vacuum service calls

* Added tests to template vacuum

* Fix indent

* Fix docstrings

* Move files for new component folder structure

* Revert additions for template_vacuum tests to common.py

* Use existing constants for template vacuum config

* Handle invalid templates

* Add tests for unused services

* Add test for invalid templates

* Fix line too long

* Do not start template change tracking in case of MATCH_ALL

* Resolve review comments

pvizeli committed:
Full speed azure

pvizeli committed:
Update azure-pipelines-ci.yml for Azure Pipelines

pvizeli committed:
Update azure-pipelines-ci.yml for Azure Pipelines

pvizeli committed:
Update azure-pipelines-ci.yml for Azure Pipelines

pvizeli committed:
Update azure-pipelines-ci.yml for Azure Pipelines

balloob committed:
Fix vacuum tests

pvizeli committed:
Update azure-pipelines-ci.yml for Azure Pipelines

balloob committed:
Make sure alert is set up after notify (#24829)

bachya committed:
Remove temperature attribute from SimpliSafe alarm control panel (#24833)

balloob committed:
Alexa sync state report (#24835)

* Do a sync after changing state reporting

* Fix entity config being None

balloob committed:
Bumped version to 0.95.3

nielstron committed:
Fronius (solar energy and inverter) component (#22316)

* Introduced fronius component that adds ability to track Fronius devices from Home Assistant

* Use device parameter for fetching inverter data

* Fixed handling of default scope

* Handle exceptions from yield

* Fulfill PR requirements

* Fixed houndci violations

* Found the last hound violation

* Fixed docstring (https://github.com/home-assistant/home-assistant/pull/11446#discussion_r165776934)

* Fixed import order with isort (https://github.com/home-assistant/home-assistant/pull/11446#discussion_r165776957)

* CONF_DEVICE is now CONF_DEVICEID (https://github.com/home-assistant/home-assistant/pull/11446#discussion_r165777161)

* Added docstring to class FroniusSensor (https://github.com/home-assistant/home-assistant/pull/11446#discussion_r165777792)

* Fixed docstring for state (https://github.com/home-assistant/home-assistant/pull/11446#discussion_r165777885)

* Added/fixed docstrings (https://github.com/home-assistant/home-assistant/pull/11446#discussion_r165778108 & https://github.com/home-assistant/home-assistant/pull/11446#discussion_r165778125)

* Remove redundant log entry (https://github.com/home-assistant/home-assistant/pull/11446#discussion_r165779213)

* Fixed error message if sensor update fails (https://github.com/home-assistant/home-assistant/pull/11446#discussion_r165779435)

* Fixed error log messages (https://github.com/home-assistant/home-assistant/pull/11446#discussion_r165779751 & https://github.com/home-assistant/home-assistant/pull/11446#discussion_r165779761)

* Satisfy hound

* Handle exceptions explicit (https://github.com/home-assistant/home-assistant/pull/11446#discussion_r168940902)

* Removed unnecessary call of update (https://github.com/home-assistant/home-assistant/pull/11446#discussion_r168940894)

* The point makes the difference.

* Removed unrelated requirements

* Remove config logging (https://github.com/home-assistant/home-assistant/pull/11446#discussion_r168968748)

* Reorder and fix imports (https://github.com/home-assistant/home-assistant/pull/11446#discussion_r168968725, https://github.com/home-assistant/home-assistant/pull/11446#discussion_r168968691)

* Update fronius requirement

* Various small fixes

* Small fixes

* Formatting

* Add fronius to coverage

* New structure and formatting

* Add manifest.json

* Fix data loading

* Make pylint happy

* Fix issues

* Fix parse_attributes

* Fix docstring and platform schema

* Make use of default HA-Const config values

* Change configuration setup, introducing list of monitored conditions

* Change the structure slightly, allowing for a list of sensors

* Remove periods from logging

* Formatting

* Change name generation, use variable instead of string

* small fixes

* Update sensor.py

* Incorporate correction proposals

* Setting default device inside validation

* Move import on top and small format

* Formatting fix

* Rename validation method to _device_id_validator

balloob committed:
Guard for None entity config (#24838)

balloob committed:
Bumped version to 0.95.4

pnbruckner committed:
Add template support to template trigger's for option (#24810)

jkeljo committed:
Upgrade sisyphus-control to 2.2 (#24837)

PR #22457 added some code that used new methods in `sisyphus-control` 2.2.
Unfortunately, because of the move to manifests it was merged still depending
on 2.1.

Fixes #24834

zewelor committed:
Fix yeelight color temp getter (#24830)

* Fix yeelight color temp getter

* Remove wrong types

fabaff committed:
Upgrade luftdaten to 0.6.1 (#24842)

* Upgrade luftdaten to 0.6.0

* Upgrade luftdaten to 0.6.1

fabaff committed:
Upgrade sqlalchemy to 1.3.5 (#24844)

fabaff committed:
Upgrade numpy to 1.16.4 (#24845)

fabaff committed:
Upgrade certifi to >= 2019.6.16 (#24846)

fabaff committed:
Upgrade pyyaml to 5.1.1 (#24847)

fabaff committed:
Upgrade jinja2 to >=2.10.1 (#24851)

fabaff committed:
Upgrade psutil to 5.6.3 (#24854)

fabaff committed:
Upgrade cryptography to 2.7 (#24852)

zewelor committed:
Fix mysensors icon name (#24871)

fabaff committed:
Update praw to 6.3.1 (#23737)

* Upgrade praw to 6.3.1

* Update praw to 6.3.1

realthk committed:
Hungarian is also supported in Google Cloud TTS  (#24861)

* Hungarian is also a supported language

* Hungarian is also a supported language

* Hungarian is also a supported language

fabaff committed:
Upgrade youtube_dl to 2019.06.27 (#24875)

cereal2nd committed:
Velbus: autodiscover covers (#24877)

* Added covers to the velbus component with autodicovery, bumped python velbus version

* Fixed some pylint stuff

fabaff committed:
Upgrade importlib-metadata to 0.18 (#24848)

fabaff committed:
Upgrade bcrypt to 3.1.7 (#24850)

dmulcahey committed:
Fix traceback during ZHA device removal (#24882)

* fix device remove lifecycle
* clean up remove signal
* add guard

andrewsayre committed:
Move SmartThings imports to top (#24878)

* Move imports to top

* use lib constants

* Add missing three_axis mapping

balloob committed:
Update translations

balloob committed:
Updated frontend to 20190630.0

JeffLIrion committed:
Bump androidtv to 0.0.17 (#24886)

* Bump androidtv to 0.0.17

* Bump androidtv to 0.0.17

Danielhiversen committed:
Upgrade pytest to 5.0.0 (#24885)

* Upgrade pytest to 5.0.0

* exception message for pytest 5

zxdavb committed:
Tweak geniushub battery icons according to device state (#24798)

* tweak battery icons according to device state/availability

* tweak battery icons according to device state/availability 2

* make dt objects aware

* make dt objects aware 2

* woops - use util.dt in favour of datetime

* woops - use util.dt in favour of datetime 2

* refactor battery icon code, remove parallel_updates

cybe committed:
Support mqtt discovery topic prefix with slashes (#24840)

kevank committed:
Update tts.py (#24892)

dmulcahey committed:
go back to signals and no hard entity references (#24894)

Swamp-Ig committed:
Update IDs for rename node/value (#24646)

* Update IDs for rename node/value

* Rename devices and entities

* Improved coverage

pvizeli committed:
Add git editor / app port

pnbruckner committed:
Bump life360 package to 4.0.1 (#24905)

pvizeli committed:
Update devcontainer.json

pvizeli committed:
Update devcontainer.json

pvizeli committed:
Update devcontainer.json

amelchio committed:
Stability improvements for Sonos availability (#24880)

* Stability improvements for Sonos availability

* Handle seen reentrancy

pnbruckner committed:
Fix 'same state' monitoring in numeric_state trigger (#24910)

pnbruckner committed:
Fix 'same state' monitoring in state trigger (#24904)

pnbruckner committed:
Add missing trigger.for variable to template trigger (#24893)

kreegahbundolo committed:
Add ability to send attachments in pushover notifications (#24806)

* Added ability to send attachments in pushover notifications

* Added full name for exception to satisfy static check

* Fixed hanging indent lint problem

* Added path checking, removed import re, changed url check method to use
startswith.

* Removed argument from logging statement.

* Changed IOError to OSError, fixed logging, added logging statement.

balloob committed:
Updated frontend to 20190702.0

cgtobi committed:
Fix home coach discovery (#24902)

* Fix home coach discovery

* Update requirements file

lufton committed:
New languages that looks like supported by Google but not documented: (#24881)

* cs-CZ – Czech, Czech Republic
* el-GR – Modern Greek (1453-), Greece
* en-IN – English, India
* fi-FI – Finnish, Finland
* fil-PH – Filipino, Philippines
* hi-IN – Hindi, India
* id-ID – Indonesian, Indonesia
* vi-VN – Vietnamese, Viet Nam
Fixed regex expression to match language codes like fil-PH

dmulcahey committed:
Remove ZHA device entity (#24909)

* move availability handling to device

* update last_seen format

* add battery sensor

* fix interval

* fix battery reporting now that it is a sensor

* remove zha entities and add battery sensor

JeffLIrion committed:
Bump androidtv to 0.0.18 (#24927)

* Bump androidtv to 0.0.18

* Bump androidtv to 0.0.18

amelchio committed:
Update pysonos to 0.0.19 (#24930)

ctso committed:
Add support for Yale YRL256 lock (#24932)

gadgetchnnel committed:
Fix exclusion of routes with excl_filter (#24928)

Fix exclusion of routes with excl_filter (was including instead of excluding)

dmulcahey committed:
New scanner device tracker and ZHA device tracker support (#24584)

* initial implementation for zha device trackers

* constant

* review comments

* Revert "review comments"

This reverts commit 2130823566820dfc114dbeda08fcdf76ed47a4e7.

* rename device tracker entity

* update trackers

* raise when not implemented

* Update homeassistant/components/device_tracker/config_entry.py

Review comment

Co-Authored-By: Martin Hjelmare <marhje52@kth.se>

* move source type to base state attrs

* review comments

* review comments

* review comments

* fix super call

* fix battery and use last seen from device

* add test

* cleanup and add more to test

* cleanup post zha entity removal PR

* add tests for base entities

* rework entity tests

bachya committed:
Allow updating of via_device in device registry (#24921)

* Allow updating of via_device in device registry

* Added test

mezz64 committed:
Update pyHik library to 0.2.3 (#24957)

cgtobi committed:
Implement ADR0003 for Netatmo sensor (#24944)

* Remove configurable monitored conditions

* Only process existing modules

* Remove unused import

* Fix linter error

nielstron committed:
Fix monitoring of trays in syncthru component (#24961)

frenck committed:
Batch of Component(s) -> Integration(s) (#24972)

apeeters committed:
Add sonos.play_queue service (#24974)

* Add sonos.play_queue service

* Add SERVICE_PLAY_QUEUE import in alphabetical order

* Add queue_position parameter for sonos.play_queue service

* Move queue_position default to schema definition

slackr31337 committed:
Fix errors if rest source becomes unavailable (#24986)

* Fix errors if rest source becomes unavailable

* Remove exclamation mark

PaulAnnekov committed:
Switched from tuyapy to tuyaha as 1st one is not maintained (#24821)

frenck committed:
Adds Lock Threads Probot (#24984)

frenck committed:
Adds Stale Probot for issues (#24985)

* Adds Stale Probot for issues

* Do not ignore assigned issues

* Small language tweak in mark comment

scop committed:
Upgrade mypy to 0.711, drop no longer needed workarounds (#24998)

https://mypy-lang.blogspot.com/2019/06/mypy-0711-released.html

Swamp-Ig committed:
Changes as per code review of #24646 (#24917)

qypea committed:
Sleepiq single sleeper crash (#24941)

* Update sleepyq to 0.7

Fixes crash when working with a single sleeper.

* sleepiq: Handle null side definitions

These happen if no sleeper is defined for a side of the bed. Don't
create sensors for null sides; they'll crash every time we try to use
them.

* sleepiq: Fix urls mocked to match sleepyq 0.7

* sleepi: Fix test_sensor.TestSleepIQSensorSetup

Sleepyq 0.7 throws on empty strings, so we have to specify them.

* sleepiq: Test for ValueError thrown by sleepyq 0.7

* sleepiq: Drop no longer used HTTPError import

* sleepiq: Add tests for single sleeper case

* sleepiq: Shorten comments to not overflow line length

* sleepiq: Use formatted string literals for adding suffixes to test files

* sleepiq: Use str.format() for test suffixing

davet2001 committed:
Add support for aurora ABB Powerone solar photovoltaic inverter (#24809)

* Add support for aurora ABB Powerone solar photovoltaic inverter

* Add support for aurora ABB Powerone solar photovoltaic inverter

* Update stale docstring

* Fixed whitespace lint errors

* Remove test code

* Delete README.md

Website documentation contains setup instructions.  README not needed here.

* Only close the serial line once.

* Correct newlines between imports

* Change add_devices to add_entites and remove unnecessary logging.

* Use new style string formatting instead of concatenation

* Directly access variables rather than via config.get

* Update sensor.py

scop committed:
Test dependency updates (#25004)

* Upgrade pytest to 5.0.1

https://docs.pytest.org/en/latest/changelog.html#pytest-5-0-1-2019-07-04

* Upgrade asynctest to 0.13.0

* Upgrade requests_mock to 1.6.0

Danielhiversen committed:
upgrade switchmate to latest lib (#25006)

tsvi committed:
Upgrade hdate==0.8.8 (#25008)

This should fix incosistencies between issur_melacha_in_effect sensor and candle_lighting time.

Probably fixes #24479 and #23852

teharris1 committed:
Upgrade insteonplm to 0.16.0 and add INSTEON scene triggering (#24765)

* Upgrade insteonplm to 0.16.0 and add INSTEON scene triggering

* Fix spacing issue

* Dummy commit to trigger CLA

* Remove dummy change

* Code review changes

* Use ENTITY_MATCH_ALL keyword from const and lint cleanup

* Make entity method print_aldb private

ludeeus committed:
Install requirements for integrations in packages before importing them. (#25005)

* Process requirements for integrations in packages before loading

* trigger buld

cadavre committed:
Added missing yeelight models mapping (#24963)

elupus committed:
Correct socket use in cert_expiry platform (#25011)

* Make sure we use same family for ssl socket and connection

getaddrinfo result could be different from what connection
was made with. It also blocks potential use of
happy eye balls algorithm

This also fixes lingering sockets until python garbage
collection.

* Add availability value if unable to get expiry

* Fix lint issue

pvizeli committed:
Climate 1.0 (#23899)

* Climate 1.0 / part 1/2/3

* fix flake

* Lint

* Update Google Assistant

* ambiclimate to climate 1.0 (#24911)

* Fix Alexa

* Lint

* Migrate zhong_hong

* Migrate tuya

* Migrate honeywell to new climate schema (#24257)

* Update one

* Fix model climate v2

* Cleanup p4

* Add comfort hold mode

* Fix old code

* Update homeassistant/components/climate/__init__.py

Co-Authored-By: Paulus Schoutsen <paulus@home-assistant.io>

* Update homeassistant/components/climate/const.py

Co-Authored-By: Paulus Schoutsen <paulus@home-assistant.io>

* First renaming

* Rename operation to hvac for paulus

* Rename hold mode to preset mode

* Cleanup & update comments

* Remove on/off

* Fix supported feature count

* Update services

* Update demo

* Fix tests & use current_hvac

* Update comment

* Fix tests & add typing

* Add more typing

* Update modes

* Fix tests

* Cleanup low/high with range

* Update homematic part 1

* Finish homematic

* Fix lint

* fix hm mapping

* Support simple devices

* convert lcn

* migrate oem

* Fix xs1

* update hive

* update mil

* Update toon

* migrate deconz

* cleanup

* update tesla

* Fix lint

* Fix vera

* Migrate zwave

* Migrate velbus

* Cleanup humity feature

* Cleanup

* Migrate wink

* migrate dyson

* Fix current hvac

* Renaming

* Fix lint

* Migrate tfiac

* migrate tado

* Fix PRESET can be None

* apply PR#23913 from dev

* remove EU component, etc.

* remove EU component, etc.

* ready to test now

* de-linted

* some tweaks

* de-lint

* better handling of edge cases

* delint

* fix set_mode typos

* apply PR#23913 from dev

* remove EU component, etc.

* ready to test now

* de-linted

* some tweaks

* de-lint

* better handling of edge cases

* delint

* fix set_mode typos

* delint, move debug code

* away preset now working

* code tidy-up

* code tidy-up 2

* code tidy-up 3

* address issues #18932, #15063

* address issues #18932, #15063 - 2/2

* refactor MODE_AUTO to MODE_HEAT_COOL and use F not C

* add low/high to set_temp

* add low/high to set_temp 2

* add low/high to set_temp - delint

* run HA scripts

* port changes from PR #24402

* manual rebase

* manual rebase 2

* delint

* minor change

* remove SUPPORT_HVAC_ACTION

* Migrate radiotherm

* Convert touchline

* Migrate flexit

* Migrate nuheat

* Migrate maxcube

* Fix names maxcube const

* Migrate proliphix

* Migrate heatmiser

* Migrate fritzbox

* Migrate opentherm_gw

* Migrate venstar

* Migrate daikin

* Migrate modbus

* Fix elif

* Migrate Homematic IP Cloud to climate-1.0 (#24913)

* hmip climate fix

* Update hvac_mode and preset_mode

* fix lint

* Fix lint

* Migrate generic_thermostat

* Migrate incomfort to new climate schema (#24915)

* initial commit

* Update climate.py

* Migrate eq3btsmart

* Lint

* cleanup PRESET_MANUAL

* Migrate ecobee

* No conditional features

* KNX: Migrate climate component to new climate platform (#24931)

* Migrate climate component

* Remove unused code

* Corrected line length

* Lint

* Lint

* fix tests

* Fix value

* Migrate geniushub to new climate schema (#24191)

* Update one

* Fix model climate v2

* Cleanup p4

* Add comfort hold mode

* Fix old code

* Update homeassistant/components/climate/__init__.py

Co-Authored-By: Paulus Schoutsen <paulus@home-assistant.io>

* Update homeassistant/components/climate/const.py

Co-Authored-By: Paulus Schoutsen <paulus@home-assistant.io>

* First renaming

* Rename operation to hvac for paulus

* Rename hold mode to preset mode

* Cleanup & update comments

* Remove on/off

* Fix supported feature count

* Update services

* Update demo

* Fix tests & use current_hvac

* Update comment

* Fix tests & add typing

* Add more typing

* Update modes

* Fix tests

* Cleanup low/high with range

* Update homematic part 1

* Finish homematic

* Fix lint

* fix hm mapping

* Support simple devices

* convert lcn

* migrate oem

* Fix xs1

* update hive

* update mil

* Update toon

* migrate deconz

* cleanup

* update tesla

* Fix lint

* Fix vera

* Migrate zwave

* Migrate velbus

* Cleanup humity feature

* Cleanup

* Migrate wink

* migrate dyson

* Fix current hvac

* Renaming

* Fix lint

* Migrate tfiac

* migrate tado

* delinted

* delinted

* use latest client

* clean up mappings

* clean up mappings

* add duration to set_temperature

* add duration to set_temperature

* manual rebase

* tweak

* fix regression

* small fix

* fix rebase mixup

* address comments

* finish refactor

* fix regression

* tweak type hints

* delint

* manual rebase

* WIP: Fixes for honeywell migration to climate-1.0 (#24938)

* add type hints

* code tidy-up

* Fixes for incomfort migration to climate-1.0 (#24936)

* delint type hints

* no async unless await

* revert: no async unless await

* revert: no async unless await 2

* delint

* fix typo

* Fix homekit_controller on climate-1.0 (#24948)

* Fix tests on climate-1.0 branch

* As part of climate-1.0, make state return the heating-cooling.current characteristic

* Fixes from review

* lint

* Fix imports

* Migrate stibel_eltron

* Fix lint

* Migrate coolmaster to climate 1.0 (#24967)

* Migrate coolmaster to climate 1.0

* fix lint errors

* More lint fixes

* Fix demo to work with UI

* Migrate spider

* Demo update

* Updated frontend to 20190705.0

* Fix boost mode (#24980)

* Prepare Netatmo for climate 1.0 (#24973)

* Migration Netatmo

* Address comments

* Update climate.py

* Migrate ephember

* Migrate Sensibo

* Implemented review comments (#24942)

* Migrate ESPHome

* Migrate MQTT

* Migrate Nest

* Migrate melissa

* Initial/partial migration of ST

* Migrate ST

* Remove Away mode (#24995)

* Migrate evohome, cache access tokens (#24491)

* add water_heater, add storage - initial commit

* add water_heater, add storage - initial commit

delint

add missing code

desiderata

update honeywell client library & CODEOWNER

add auth_tokens code, refactor & delint

refactor for broker

delint

* Add Broker, Water Heater & Refactor

add missing code

desiderata

* update honeywell client library & CODEOWNER

add auth_tokens code, refactor & delint

refactor for broker

* bugfix - loc_idx may not be 0

more refactor - ensure pure async

more refactoring

appears all r/o attributes are working

tweak precsion, DHW & delint

remove unused code

remove unused code 2

remove unused code, refactor _save_auth_tokens()

* support RoundThermostat

bugfix opmode, switch to util.dt, add until=1h

revert breaking change

* store at_expires as naive UTC

remove debug code

delint

tidy up exception handling

delint

add water_heater, add storage - initial commit

delint

add missing code

desiderata

update honeywell client library & CODEOWNER

add auth_tokens code, refactor & delint

refactor for broker

add water_heater, add storage - initial commit

delint

add missing code

desiderata

update honeywell client library & CODEOWNER

add auth_tokens code, refactor & delint

refactor for broker

delint

bugfix - loc_idx may not be 0

more refactor - ensure pure async

more refactoring

appears all r/o attributes are working

tweak precsion, DHW & delint

remove unused code

remove unused code 2

remove unused code, refactor _save_auth_tokens()

support RoundThermostat

bugfix opmode, switch to util.dt, add until=1h

revert breaking change

store at_expires as naive UTC

remove debug code

delint

tidy up exception handling

delint

* update CODEOWNERS

* fix regression

* fix requirements

* migrate to climate-1.0

* tweaking

* de-lint

* TCS working? & delint

* tweaking

* TCS code finalised

* remove available() logic

* refactor _switchpoints()

* tidy up switchpoint code

* tweak

* teaking device_state_attributes

* some refactoring

* move PRESET_CUSTOM back to evohome

* move CONF_ACCESS_TOKEN_EXPIRES CONF_REFRESH_TOKEN back to evohome

* refactor SP code and dt conversion

* delinted

* delinted

* remove water_heater

* fix regression

* Migrate homekit

* Cleanup away mode

* Fix tests

* add helpers

* fix tests melissa

* Fix nehueat

* fix zwave

* add more tests

* fix deconz

* Fix climate test emulate_hue

* fix tests

* fix dyson tests

* fix demo with new layout

* fix honeywell

* Switch homekit_controller to use HVAC_MODE_HEAT_COOL instead of HVAC_MODE_AUTO (#25009)

* Lint

* PyLint

* Pylint

* fix fritzbox tests

* Fix google

* Fix all tests

* Fix lint

* Fix auto for homekit like controler

* Fix lint

* fix lint

Chris-Johnston committed:
Implement Twilio SMS notify MediaUrl support (#24971)

* Implement Twilio SMS notify MediaUrl support

Adds support for setting the `media_url` parameter of the twilio API
client with an optional attribute under the notify `data`
attribute.

Per the twilio docs (https://www.twilio.com/docs/sms/send-messages#include-medi$
this feature is only available in the US and Canada, for
GIF, PNG, or JPEG content.

* lint: fix 80 char ruler

* use kwargs to set the media_url

after testing locally, seems like the previous way of using
object() was not working. this seems to be working

* re-use the ATTR_MEDIAURL attribute

jesserizzo committed:
Enphase envoy individual inverter production (#24445)

* bump envoy_reader version to 0.4

* bump dependency envoy_reader to 0.4

* Enphase envoy get individual inverter production

* Add period in function description

* Fix dumb typo

* Define _attributes in __init__

* Better error messages, make update async

* Fix format error

* Fix pylint errors

* set unknown state to None

* Bump envoy_reader version to 0.8

* Change attributes to separate sensors

* Fix dumb thing

* Improve platform_setup for inverters

* Remove unneeded self._attributes, refactor platform setup

* Refactor platform setup

elupus committed:
Add support for arcam fmj receivers (#24621)

* Add arcam_fmj support

* Just use use state in player avoid direct client access

* Avoid leaking exceptions on invalid data

* Fix return value for volume in case of 0

* Mark component as having no coverage

* Add new requirement

* Add myself as maintainer

* Correct linting errors

* Use async_create_task instead of async_add_job

* Use new style string format instead of concat

* Don't call init of base class without init

* Annotate callbacks with @callback

Otherwise they won't be called in loop

* Reduce log level to debug

* Use async_timeout instead of wait_for

* Bump to version of arcam_fmj supporting 3.5

* Fix extra spaces

* Drop somewhat flaky unique_id

* Un-blackify ident to satisy pylint

* Un-blackify ident to satisy pylint

* Move default name calculation to config validation

* Add test folder

* Drop unused code

* Add tests for config flow import

Danielhiversen committed:
Sensibo, add HVAC_MODE_OFF (#25016)

balloob committed:
Fix ecobee flaky test (#25019)

balloob committed:
Fix mimetypes on borked Windows machines (#25018)

pnbruckner committed:
Add datetime option to input_datetime.set_datetime service (#24975)

pnbruckner committed:
Add template support to numeric_state trigger's for option (#24955)

pnbruckner committed:
Add template support to state trigger's for option (#24912)

balloob committed:
Fix Nest sensor (#25023)

Danielhiversen committed:
ambiclimate hvac_modes (#25015)

* ambiclimate hvac_modes

* style

pvizeli committed:
homematic add off support for climate (#25017)

* homematic add off support for climate

* fix lint

elupus committed:
Support config flow on custom components (#24946)

* Support populating list of flows from custom components

* Re-allow custom component config flows

* Add tests for custom component retrieval

* Don't crash view if no handler exist

* Use get_custom_components instead fo resolve_from_root

* Switch to using an event instead of lock

* Leave list of integrations as set

* The returned list is not guaranteed to be ordered

Backend uses a set to represent them.

andrewsayre committed:
Improve SmartThings test mocking (#25028)

* Migrate to asynctest

* Simplify mock access

* Use mocks

frenck committed:
Toon fixes for Climate 1.0 (#25027)

andrewsayre committed:
Add hvac fan state (#25030)

bachya committed:
Add support for Notion Home Monitoring (#24634)

* Add support for Notion Home Monitoring

* Updated coverage

* Removed auto-generated translations

* Stale docstrings

* Corrected hardware version

* Fixed binary sensor representation

* Cleanup and update protection

* Updated log message

* Cleaned up is_on

* Updated docstring

* Modified which data is updated during async_update

* Added more checks during update

* More cleanup

* Fixed unhandled exception

* Owner-requested changes (round 1)

* Fixed incorrect scan interval retrieval

* Ugh

* Removed unnecessary import

* Simplified everything via dict lookups

* Ensure bridges are properly registered

* Fixed tests

* Added catch for invalid credentials

* Ensure bridge ID is updated as necessary

* Updated method name

* Simplified bridge update

* Add support for updating bridge via_device_id

* Device update guard clause

* Removed excess whitespace

* Whitespace

* Owner comments

* Member comments

cgtobi committed:
Fix Netatmo climate presets (#25029)

* Fix netatmo presets

* Remove off mode for valves

* Revert usage of global const

* Flip values

* Remove try...except block

arigilder committed:
Add missing support for jewish_calendar.omer_count sensor (#24958)

* Add missing support for omer_count to jewish_calendar

* Add tests for omer sensor

* Add tests for omer after tzeit hakochavim

* Lint fixes

frenck committed:
Centralizes Toon data, reducing API calls (#23988)

* Centralizes Toon data, reducing API calls

Fixes #21825

Signed-off-by: Franck Nijhof <frenck@addons.community>

* Fixes bad copy past action in services.yaml

Signed-off-by: Franck Nijhof <frenck@addons.community>

* Addresses review comments

Signed-off-by: Franck Nijhof <frenck@addons.community>

* :shirt: Fixes too many blank lines

* Unsub dispatcher

pvizeli committed:
Update build pipeline

fabaff committed:
Upgrade youtube_dl to 2019.07.02 (#24990)

* Upgrade youtube_dl to 2019.07.01

* Update homeassistant/components/media_extractor/manifest.json

Co-Authored-By: Josef Schlehofer <pepe.schlehofer@gmail.com>

* Update requirements_all.txt

Co-Authored-By: Josef Schlehofer <pepe.schlehofer@gmail.com>

exxamalte committed:
updated geojson_client library to version 0.4 (#25039)

frenck committed:
Improve toon climate (#25040)

* Renames internal climate state variable to preset

* Shorten function comments

* Updates local variables on preset and temp changes

* Adds support for hvac_action

cgtobi committed:
Fix Netatmo rain gauge precision (#25036)

jlrgraham committed:
Bump pyvera to 0.3.2, null/missing value protection (#25041)

* Bump pyvera to 0.3.2, null/missing value protection.

* Add another place where the pyvera version is set.

zombielinux committed:
Add radiotherm CT80 current humidity support (#25024)

* Added CT80 Current Humidity Support

Added a check for if device is a CT80, and if so, queries the humidity object to get the current measured humidity reading.

* Update climate.py

Removed whitespace on line 229

* Update climate.py

Added humidity property. Version on local machine had that from previous tinkering.

* Update climate.py

Removed whitespace

* Update climate.py

Fixed tstat error handling for humidity data.

bachya committed:
Add config flow support to Geolocation (#25046)

PaulAnnekov committed:
Update tuyaha to 0.0.2 to catch API exceptions (#25050)

* Update tuyaha to 0.0.2 to catch API exceptions

* Updated tuyaha version in requirements

Matte23 committed:
Added marker sensor to CUPS integration (#25037)

cdce8p committed:
Remove myself as codeowner (#25043)

zxdavb committed:
[climate] Bugfix honeywell misleading error message (#25048)

* initial commit

* refactor for sync

* minor tweak

* refactor convert code

* fix regression

* remove bad await

* de-lint

* de-lint 2

* improve error message

* rebase

* tweak

* de-lint

amelchio committed:
Add Sonos debug logging (#25063)

balloob committed:
Updated frontend to 20190710.0

balloob committed:
Update translations

Swamp-Ig committed:
Avoid flooding steam API (#23941)

kellerza committed:
Add check_config helper (#24557)

* check_config

* no ignore

* tests

* try tests again

dmulcahey committed:
bump zha-quirks version (#25059)

balloob committed:
Fix broken test in Python 3.7 (#25067)

amelchio committed:
Fix for Sonos debug logging (#25064)

* Fix for Sonos debug logging

* Start logging messages with capital letters

Adminiuga committed:
Bump up ZHA dependencies. (#25062)

Bump zigpy-homeassistant to 0.7.0
Bump zigpy-deconz to 0.2.1
Bump zigpy-xbee-homeassistant to 0.4.0

bachya committed:
Add support for World Wide Lightning Location Network (#25001)

* Add support for World Wide Lightning Location Network

* Updated .coveragerc

* Added test

* Updated requirements

* Fixed tests

* Use local time for nearest strike

* Base geo location in place

* Finished geolocation work

* Fixed tests

* Cleanup

* Removed no-longer-needed method

* Updated requirements

* Add support for window and attrs

* Add strike ID to entity name

* Member comments

kellerza committed:
SMA catch error (#25045)

* SMA small fix

* lib update

* req

Martijn02 committed:
Change phrases in the logbook component for persons and binary_sensors (#25053)

Persons are now threated the same as device trackers, so the logbook states
"<name> is at <location>" or "<name> is away" instead of "<name> changed to <location|not_home>"

Binary sensors now show phrases that relate to their device_class attribute.
So "Front door is closed" instead of "Front door turned off" or "Hallway PIR detected movement"
instead of "Hallway PIR turned on"

JeffLIrion committed:
Make sure volume level is valid when incrementing/decrementing (#25061)

* Make sure volume level is not None before incrementing/decrementing

* Pass linting checks

monte-monte committed:
Complete OPERATION_MODES (#25069)

XKNX library has complete list of KNX controller modes, but current version of HA KNX climate plugin uses only two of them and one is named incorrectly ("Dehumidification" instead of "Dry"). https://github.com/XKNX/xknx/blob/master/xknx/knx/dpt_hvac_mode.py
I've added missing control modes, which has corresponding operation mode in HA. Tested this patch on my KNX IntesisBox which is used with Mitsubishi split AC, all modes were detected correctly and working as expected.
I've also corrected datapoint number in a comment, because it was pointing to a wrong one: http://www.sti.uniurb.it/romanell/Domotica_e_Edifici_Intelligenti/110504-Lez10a-KNX-Datapoint%20Types%20v1.5.00%20AS.pdf see page 94.

cgarwood committed:
Add websocket API command for Z-Wave network status (#25066)

* Add websocket API command for Z-Wave network status

* lint

* Add callback decorator

* Remove state_str, fix lint

balloob committed:
pylint

balloob committed:
Correctly store removed entities for restore state (#25073)

* Correctly store removed entities for restore state

* Lint

* Do not assume about set encoding

pnbruckner committed:
Restore automation last_triggered as datetime & fix test (#24951)

* Restore automation last_triggered as datetime & fix test

* last_triggered is always a string

balloob committed:
Version bump to 0.97.0dev0

MartinHjelmare committed:
Rewrite calendar component (#24950)

* Correct google calendar test name

* Rewrite calendar component

* Save component in hass.data.
* Rename device_state_attributes to state_attributes.
* Remove offset attribute from base state_attributes.
* Extract offset helpers to calendar component.
* Clean imports.
* Remove stale constants.
* Remove name and add async_get_events.
* Add normalize_event helper function. Copied from #21495.
* Add event property to base entity.
* Use event property for calendar state.
* Ensure event start and end.
* Remove entity init.
* Add comment about event data class.
* Temporary keep old start and end datetime format.

* Convert demo calendar

* Convert google calendar

* Convert google calendar.
* Clean up google component.
* Keep offset feature by using offset helpers.

* Convert caldav calendar

* Clean up caldav calendar.
* Update caldav cal on addition.
* Bring back offset to caldav calendar.
* Copy caldav event on update.

* Convert todoist calendar

bachya committed:
Change unique_id formula for Notion entities (#25076)

* Change unique_id formula for Notion entities

* Don't use name

balloob committed:
Do not reverse open/close calls (#24879)

pvizeli committed:
Update azure-pipelines-release.yml for Azure Pipelines

pvizeli committed:
Update azure-pipelines-release.yml for Azure Pipelines

pvizeli committed:
Update azure-pipelines-release.yml for Azure Pipelines

balloob committed:
Guard module being None (#25077)

nielstron committed:
Remove monitored conditions from syncthru (#25052)

pvizeli committed:
Update azure-pipelines-wheels.yml for Azure Pipelines

SukramJ committed:
Add climate related services to Homematic IP Cloud (#25079)

* add hmip climate services

* Rename accesspoint_id to hapid

to comply with config

* Revert "Rename accesspoint_id to hapid"

This reverts commit 4a3cd14e1482fb508273c728ad8020945b02e426.

MartinHjelmare reviewed (changes requested):
`requirements_all.txt` is not up to date.
Please run `script/gen_requirements_all.py`.

pvizeli committed:
Fix powercontrol media player alexa (#25080)

farmio committed:
Update KNX component to xknx 0.11 (#24738)

* update component for xknx 0.11.0

- expose sensor state is not casted to float anymore
- climate mode operation list has no more None values
- light supports white_value (rgbw)
- sensor expects `group_address_state` now instead of `group_address`
- sensor forwards device_class if available

* update manifest to use xknx 0.11.0

* update requirements_all for xknx 0.11.0

* update for xknx 0.11.1

- require xknx 0.11.1
- use 'state_address' instead of 'address' in sensor and binary_sensor configuration
- optional 'sync_state' for sensors and binary_sensors

* remove questionable `del kwargs`

pvizeli committed:
Update azure-pipelines-wheels.yml

pvizeli committed:
Add support for on/off climate (#25026)

* Add support for on/off climate

* address comments

* Add test for sync overwrite

* Add more tests

amelchio committed:
Support podcast episodes as Sonos favorites (#25087)

MartinHjelmare commented:
Please rebase on latest dev branch to let the build pass.

pvizeli committed:
Support hass-release inside devcontainer (#25090)

pvizeli committed:
Update azure-pipelines-ci.yml for Azure Pipelines

Commit:
Update requirements_all

depl0y commented:
@MartinHjelmare I am not sure I did this right...

MartinHjelmare commented:
No, it's a bit harder to rebase when you're not on a separate feature branch but on the same branch as the main branch. Take for habit to work on feature branches with different name than the main branch.

You can open a new PR and start over if you prefer not to take the scenic route in the git adventures. Ping me from that PR and I'll approve. :+1:

depl0y commented:
@MartinHjelmare Will do, sounds a lot better. Closing this one
